foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -1377,77 +848,22 @@ Application developers can use the `setLogLevel` method to filter the log messag
 From least to most verbose, the order of Mobile SDK logging modes is as follows:
 
 * ERROR
-* WARNING
-* DEBUG
-* VERBOSE / TRACE 
-
-When debugging on iOS, you can use `LogLevel.verbose` to enable all the logging messages that are coming from Mobile SDK and partner extensions. Similarly, on Android, you can use `LoggingMode.VERBOSE` to enable all the logging messages that are coming from Mobile SDK and partner extensions.
-
-In a production application, you should use a less verbose logging mode, such as error-level logging.
-
-By default, Mobile SDK logging mode is set to `LoggingMode.ERROR` for Android and `LogLevel.error`on iOS.
-
-{% hint style="info" %}
-* On **Android**, Mobile SDK uses the `android.util.Log` class to log messages.
-* On **iOS**, Mobile SDK uses `NSLog` to messages to Apple System Log facility.
-{% endhint %}
-
-{% tabs %}
-{% tab title="Android" %}
-**Java**
-
-**Syntax**
-
-```java
-public static void setLogLevel(LoggingMode mode)
-```
-
-**Example**
-
-```java
-import com.adobe.marketing.mobile.LoggingMode;
-import com.adobe.marketing.mobile.MobileCore;
-
-MobileCore.setLogLevel(LoggingMode.VERBOSE);
-```
-{% endtab %}
-
-{% tab title="iOS \(AEP 3.x\)" %}
-**Swift**
-
-**Syntax**
-
-```swift
- public static func setLogLevel(_ level: LogLevel)
-```
-
-**Example**
-
-```swift
-import AEPCore
-import AEPServices
-
-  MobileCore.setLogLevel(.trace)
-```
-
-**Objective C**
+* WARNING
+* DEBUG
+* VERBOSE / TRACE 
 
-**Syntax**
+When debugging on iOS, you can use `LogLevel.verbose` to enable all the logging messages that are coming from Mobile SDK and partner extensions. Similarly, on Android, you can use `LoggingMode.VERBOSE` to enable all the logging messages that are coming from Mobile SDK and partner extensions.
 
-```swift
- @objc(setLogLevel:)
- public static func setLogLevel(_ level: LogLevel)
-```
+In a production application, you should use a less verbose logging mode, such as error-level logging.
 
-**Example**
+By default, Mobile SDK logging mode is set to `LoggingMode.ERROR` for Android and `LogLevel.error`on iOS.
 
-```objectivec
-@import AEPCore;
-@import AEPServices;
+{% hint style="info" %}
+* On **Android**, Mobile SDK uses the `android.util.Log` class to log messages.
+* On **iOS**, Mobile SDK uses `NSLog` to messages to Apple System Log facility.
```

Comment:
nit: I'm assuming we want to retain the Android info in the hint here for wrapper purposes. Also, iOS info needs correction `On **iOS**, Mobile SDK uses `NSLog` to send messages to Apple System Log facility.`

Reply:
In general I kept the top-level description and removed specific tabs, I think it makes sense to keep it here for info purposes as the wrapper calls the native implementation.